lib/utils.js

```diff
@@ -0,0 +1,11 @@
+var path = require('path');
+
+/* Normalize URL or path to an absolute path, if it corresponds to one. */
+function filePath(urlOrPath) {
+  // give naked paths the benefit of the doubt that they are file paths
+  if (path.isAbsolute(urlOrPath))
```

Comment:
`path.isAbsolute('http://site.com') === false`, `path` is specifically for file-system paths. Perhaps you meant to use URL?